Jenkinsfile

```diff
@@ -1,27 +1,78 @@
+@Library('pipeline-library') _
 pipeline {
+    agent {
+        node{
+            label "dev_mesos_large_v2"
+        }
+    }
+
+    environment {
+        CREDENTIALS_ID  = "GENESYSCLOUD_OAUTHCLIENT_ID_AND_SECRET"
+        GOPATH = "$HOME/go"
+		TF_ACC = "1"
+        TF_LOG = "DEBUG"
+        TF_LOG_PATH = "../test.log"
+		GENESYSCLOUD_REGION = "us-east-1"
+        GENESYSCLOUD_SDK_DEBUG =  "true"
+        GENESYSCLOUD_TOKEN_POOL_SIZE =  20
+    }
+    tools {
+        go 'Go 1.20'
```

Comment:
We are using go 1.21 in the provider, should we use the same here?

Reply:
1.21 isn't available as of yet in Jenkins.